bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -73,13 +76,23 @@
         <li id="{{ forum.dashed }}">
           <div class="summary">
             <h4>{{ forum.id }}<a href="#{{ forum.dashed }}">#</a></h4>
-            <p class="description">{{ forum.description|safe }}</p>
+            <p class="description">
+              {%- if forum.id in mdn_forums -%}
```

Comment:
Curious why we're putting all this special casing in the template for this. Is there any reason why we can't just update [the source](https://github.com/mozilla/community-data/blob/master/forums/raw-ng-list.txt)?